ui/changes/src/index.js

```diff
@@ -233,6 +237,35 @@ function addRow(bugSummary) {
     risk_text.style.color = HIGH_RISK_COLOR;
     risk_text.textContent = 'Higher';
   }
+}
+
+async function populateVersions() {
+  var versionSelector = document.getElementById("releaseVersion")
+
+  let data = await landingsData;
+
+  var allVersions = new Set();
+    for (let date in data) {
+      for (let bug of data[date]) {
```

Comment:
Nit: you could avoid the outer for loop by doing:
```
for (let bugSummary of Object.values(data)) {
```

Reply:
Oh that's nice. I just defaulted to basics as my JS is still rusty 😅

Reply:
You could even use `reduce`, something like this:
```JavaScript
let versions = [
  ...Object.values(data)
    .filter((bugSummary) => bugSummary.versions.length > 0)
    .reduce((set, bugSummary) => set.add(...bug.versions), new Set())
].sort();
```

It doesn't matter much in this case, but sometimes I find the functional style more readable.